autoware_map_msgs/srv/GetSelectedLanelet2Map.srv

```diff
@@ -0,0 +1,9 @@
+# Osm file ids which are selected to be loaded
+int32[] vector_map_file_ids
```

Comment:
https://github.com/autowarefoundation/autoware_msgs/blob/main/autoware_map_msgs/srv/GetSelectedPointCloudMap.srv#L2

Can you do it similarly to this? :pray: 
```suggestion
string[] cell_ids
```

Reply:
updated